runtime/src/lib.rs

```diff
@@ -211,7 +211,9 @@ parameter_types! {
 
     pub BlockLength: frame_system::limits::BlockLength = frame_system::limits::BlockLength
         ::max_with_normal_ratio(5 * 1024 * 1024, NORMAL_DISPATCH_RATIO);
-    pub const SS58Prefix: u8 = 42;
+
+    // ASCII for ZKV
```

Comment:
Maybe a bit pedantic but it's not immediate to get this explanation: I would rewrite `// ASCII for 'Z'+'K'+'V'`.
But just my taste, feel free to ignore.